wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -177,10 +175,11 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 #### Guidelines
 
 - **Hyperdashes should only be used on drops and/or slider repetitions when the slider path is simple and easy-to-follow.** This is to prevent chaotic or unreasonably difficult slider movement, as transitioning into and out of complex slider shapes with hyperdash is usually uncomfortable to play and a major penalty to accuracy for little benefit.
-- Combos should not exceed 16 objects including slider tails and repeats. Spinners are an exception.
-- Note density should follow a mostly 1/2 + 1/4 and/or 1/3 + 1/6 pattern - 1/8 patterns and higher can be used sparingly.
 - 1/8 and higher hyperdashes should not be used between consecutive object pairs.
 - Edge dashes may be used with caution for a maximum of three consecutive objects, and should not be used after hyperdashes.
+- Combos should not exceed 16 objects including slider tails and repeats. Spinners are an exception.
```

Comment:
again i think "spinners" is an osu!standard word, unless if that's way more common than saying bananas or showers idk

Reply:
yeah it's still a spinner in catch

Reply:
Spinners is fine. It's the editor term.